From c9bdbe50dea19669a1248f29fbc61bf08953aca6 Mon Sep 17 00:00:00 2001
From: Roland Haas <rhaas@tapir.caltech.edu>
Date: Tue, 12 Nov 2013 17:56:03 -0800
Subject: [PATCH 07/18] EOS_Omni: remove unnecessary make.code.deps

Cactus tracks dependencies on include files on its own using calls to
the C compiler
---
 src/make.code.deps             |  1 -
 src/nuc_eos_cxx/make.code.deps | 18 ------------------
 2 files changed, 19 deletions(-)
 delete mode 100644 src/make.code.deps
 delete mode 100644 src/nuc_eos_cxx/make.code.deps

diff --git a/src/make.code.deps b/src/make.code.deps
deleted file mode 100644
index cd5bfde..0000000
--- a/src/make.code.deps
+++ /dev/null
@@ -1 +0,0 @@
-EOS_Omni_Startup.F90:	EOS_Omni_Module.o
diff --git a/src/nuc_eos_cxx/make.code.deps b/src/nuc_eos_cxx/make.code.deps
deleted file mode 100644
index 9e26b00..0000000
--- a/src/nuc_eos_cxx/make.code.deps
+++ /dev/null
@@ -1,18 +0,0 @@
-nuc_eos_dpdrhoe_dpderho.cc:	nuc_eos.hh 
-nuc_eos.cc:			nuc_eos.hh 
-nuc_eos_press.cc:		nuc_eos.hh
-nuc_eos_press_cs2.cc:		nuc_eos.hh
-nuc_eos_short.cc:		nuc_eos.hh
-nuc_eos_full.cc:		nuc_eos.hh
-readtable.cc:			nuc_eos.hh
-nuc_eos_dpdrhoe_dpderho.cc:	helpers.hh 
-nuc_eos.cc:			helpers.hh 
-nuc_eos_press.cc:		helpers.hh
-nuc_eos_press_cs2.cc:		helpers.hh
-nuc_eos_short.cc:		helpers.hh
-nuc_eos_full.cc:		helpers.hh
-readtable.cc:			helpers.hh
-
-
-
-
-- 
1.8.5.3

